### PR TITLE
feat: publish SDK7 scenes

### PR DESCRIPTION
**Pull request by cazala**
This PR adds support to publish SDK7 scenes to LAND and to Worlds

### CODE REVIEW

File: src/modules/deployment/sagas.ts [hunk 1]
```diff
@@ -192,6 +193,8 @@ export function* deploymentSaga(builder: BuilderAPI, catalystClient: CatalystCli
       // Remove the old communications property if it exists
       const sceneDefinition: SceneDefinition = JSON.parse(files[EXPORT_PATH.SCENE_FILE])
 
+      console.log('definition', JSON.stringify(sceneDefinition, null, 2))
```
Comment (meelrossi) [inline src/modules/deployment/sagas.ts:196]: 😬

File: src/modules/deployment/sagas.ts [hunk 2]
```diff
@@ -218,9 +221,111 @@ export function* deploymentSaga(builder: BuilderAPI, catalystClient: CatalystCli
...
+        owner: address || '',
+        main: 'bin/index.js',
+        contact: {
+          name: name || '',
+          email: ''
+        },
+        display: {
+          title: project.title,
+          favicon: 'favicon_asset'
+          // navmapThumbnail: 'scene-thumbnail.png'
```
Comment (meelrossi) [inline src/modules/deployment/sagas.ts:277]: is this needed?

File: src/modules/deployment/sagas.ts [hunk 3]
```diff
@@ -218,9 +221,111 @@ export function* deploymentSaga(builder: BuilderAPI, catalystClient: CatalystCli
...
+        allowedMediaHostnames: [],
+        owner: address || '',
+        main: 'bin/index.js',
+        contact: {
+          name: name || '',
+          email: ''
+        },
+        display: {
+          title: project.title,
+          favicon: 'favicon_asset'
```
Comment (meelrossi) [inline src/modules/deployment/sagas.ts:276]: not sure if this does anything

**Review by meelrossi — APPROVED** [review]

File: src/modules/deployment/sagas.ts (hunk 2 above)
Comment (cazala) [reply src/modules/deployment/sagas.ts]: good catch! it had to be uncommented

**Pull request merged**